1Homework18.02.22/SquareSorting/SquareSorting/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+
+namespace SquareSorting;
+internal class Program
+{
+    /// <summary>
+    /// Checks if an array is sorted
+    /// </summary>
+    /// <param name="array">the sortable array</param>
+    /// <returns>true if the array is sorted</returns>
+    private static bool IsArraySorted(int[] array)
+    {
+        for (int i = 0; i < array.Length - 1; ++i)
+        {
+            if (array[i] > array[i + 1])
+            {
+                return false;
+            }
+        }
+
+        return true;
+    }
+    
+    /// <summary>
+    /// Swaps two integer elements
+    /// </summary>
+    /// <param name="firstVariable"></param>
+    /// <param name="secondVariable"></param>
+    private static void Swap(ref int firstVariable, ref int secondVariable)
+    {
+        firstVariable ^= secondVariable;
+        secondVariable ^= firstVariable;
+        firstVariable ^= secondVariable;
```

Comment:
А ещё можно вот так:
```suggestion
        (firstVariable, secondVariable) = (secondVariable, firstVariable);
```